### PR TITLE
Fix delete commands to delete associated enrolment

### LINKED ISSUE #138: Update deletion of member and events to fit Attendance

### PR DESCRIPTION
**Pull request by wr1159**
Closes #138 

What have you done in this PR?
* Created new functions in model called `deleteEnrolmentsWithEventName()` and `deleteEnrolmentsWithMemberName()`
* Update tests

Dev Testing Steps / Screenshots
First, enrol members to events and deleting the member should also delete associated enrolments. Likewise, deleting the event should also delete associated enrolments.

```
enrol m/1 e/1 h/0 r/r
enrol m/1 e/2 h/0 r/r
enrol m/2 e/1 h/0 r/r
enrol m/2 e/2 h/0 r/r
enrol m/1 e/3 h/0 r/r
enrol m/3 e/1 h/0 r/r
deleteMember 1
deleteEvent 1
```

### CODE REVIEW

File: src/main/java/seedu/ccacommander/logic/commands/DeleteMemberCommand.java [hunk 1]
```diff
@@ -33,13 +36,18 @@ public DeleteMemberCommand(Index targetIndex) {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         List<Member> lastShownList = model.getFilteredMemberList();
+        List<Enrolment> enrolmentsList = model.getFilteredEnrolmentList();
+        List<Enrolment> enrolmentsToBeDeletedList = new ArrayList<>();
```
Comment (dylkaw) [inline src/main/java/seedu/ccacommander/logic/commands/DeleteMemberCommand.java]:
```suggestion
```
Is this unused? Can it be deleted?

**Review by dylkaw — COMMENTED** [review]
Just a small nit, LGTM!

File: src/main/java/seedu/ccacommander/logic/commands/DeleteMemberCommand.java (hunk 1 above)
Comment (wr1159) [reply src/main/java/seedu/ccacommander/logic/commands/DeleteMemberCommand.java]: Yes it can

**Review by ph-nathan — APPROVED** [review]
LGTM!

**Pull request merged**